Formula/darktable.rb

```diff
@@ -0,0 +1,82 @@
+class Darktable < Formula
+  desc "Open source photography workflow application and raw developer"
```

Comment:
What do you about this description?

`Open source photography workflow and raw processing software`